src/sgame/sg_bot_ai.cpp

```diff
@@ -175,6 +196,14 @@ botEntityAndDistance_t AIEntityToGentity( gentity_t *self, AIEntity_t e )
 	{
 		return self->botMind->closestDamagedBuilding;
 	}
+	else if ( e == E_HUMANBUILDING )
```

Comment:
while you're there, maybe we'd rather see a switch, here? The only case in which it would not work nicely is when a specific buildable is required for, but the check could go in a default: statement, I guess?